### ISSUE TITLE
With manual Tomcat installation, automatic activation of a db-loaded config does not work

### ISSUE THREAD

**Issue by mhdirkse** [ibissource/iaf#841]
**Describe the bug**
I installed Apache Tomcat manually under Windows. I used a custom-build WAR file that held the Frank!Framework and a custom Java class. Then I uploaded a Frank configuration with the activate checkbox checked. The configuration did not become active in a minute. I had to go to the Adapter Status page and click the Refresh button to activate it.

**Reporter**
Martijn Dirkse

**Category**
 - Frank Framework

**To Reproduce**
1. Clone repository https://github.com/ibissource/frank-maven-examples. Checkout commit a636ec6b943f94b4a0cc820368fd45eef98789ab.
2. Follow the instructions from Frank2MultiConfig/README.md to build the WAR file with the custom class.
3. Install Apache Tomcat like explained in Frank2Maven/README.md
4. Follow the Tomcat installation instructions of Frank2MultiConfig/README.md. This should include deploying the WAR and using the upload configuration screen to upload the config of Frank2MultiConfig_config. Uploading happens with activateConfig checked.
5. Wait two minutes and go to the manage configurations page. The configuration "custom" is still red.
6. Go to the Adapter Status page. Press the Refresh button.
7. Only now, configuration "custom" is usable. Go to Test Pipeline and test its adapter "AdapterWithCustom". In the Message field, put an integer number. The result should be that number plus three.

**Expected behavior**
Step 7 should work without the need to do step 6.

**Screenshots**
Not applicable.

**Environment:**
 - Manual installation of Apache Tomcat, version 7.0.104.
 - Microsoft Edge.
 - Frank!Framework version 7.6-20200518.212846.

**Additional context**
I checked the logs and I saw the following every minute:

2020-06-17 19:30:53,940 DEBUG [scheduler_Worker-4] null simpl.SimpleThreadPool - WorkerThread is shut down.
2020-06-17 19:30:53,941 DEBUG [ibisContextReconnectThread] null simpl.SimpleThreadPool - No executing jobs remaining, all threads stopped.
2020-06-17 19:30:53,941 DEBUG [ibisContextReconnectThread] null simpl.SimpleThreadPool - Shutdown of threadpool complete.
2020-06-17 19:30:53,942 INFO  [ibisContextReconnectThread] null core.QuartzScheduler - Scheduler scheduler_$_one shutdown complete.
2020-06-17 19:30:53,942 ERROR [ibisContextReconnectThread] null configuration.IbisContext - Failed to initialize IbisContext, retrying in 1 minute!
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowGenerator' defined in class path resource [springCommon.xml]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: J2V8 native library not loaded
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1630) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:756) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867) ~[spring-context-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:542) ~[spring-context-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at nl.nn.adapterframework.lifecycle.IbisApplicationContext.createApplicationContext(IbisApplicationContext.java:118) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:121) [ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.configuration.IbisContext.init(IbisContext.java:101) [ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.configuration.IbisContextRunnable.run(IbisContextRunnable.java:25) [ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_251]
Caused by: java.lang.IllegalStateException: J2V8 native library not loaded
	at com.eclipsesource.v8.V8.checkNativeLibraryLoaded(V8.java:195) ~[j2v8_linux_x86_64-4.6.0.jar:?]
	at com.eclipsesource.v8.V8.createV8Runtime(V8.java:149) ~[j2v8_linux_x86_64-4.6.0.jar:?]
	at nl.nn.adapterframework.extensions.javascript.J2V8.startRuntime(J2V8.java:73) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine$Env.<init>(GraphvizEngine.java:169) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine.getEnv(GraphvizEngine.java:130) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine.<init>(GraphvizEngine.java:64) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine.<init>(GraphvizEngine.java:51) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.util.flow.V8FlowGenerator.afterPropertiesSet(V8FlowGenerator.java:40) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1688) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1626) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	... 14 more
Caused by: java.lang.UnsatisfiedLinkError: Could not load J2V8 library. Reasons: 
	no j2v8_win32_x86_64 in java.library.path

	at com.eclipsesource.v8.LibraryLoader.loadLibrary(LibraryLoader.java:75) ~[j2v8_linux_x86_64-4.6.0.jar:?]
	at com.eclipsesource.v8.V8.load(V8.java:71) ~[j2v8_linux_x86_64-4.6.0.jar:?]
	at com.eclipsesource.v8.V8.createV8Runtime(V8.java:145) ~[j2v8_linux_x86_64-4.6.0.jar:?]
	at nl.nn.adapterframework.extensions.javascript.J2V8.startRuntime(J2V8.java:73) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine$Env.<init>(GraphvizEngine.java:169) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine.getEnv(GraphvizEngine.java:130) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine.<init>(GraphvizEngine.java:64) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.extensions.graphviz.GraphvizEngine.<init>(GraphvizEngine.java:51) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at nl.nn.adapterframework.util.flow.V8FlowGenerator.afterPropertiesSet(V8FlowGenerator.java:40) ~[ibis-adapterframework-core-7.6-20200518.212846.jar:7.6-20200518.212846]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1688) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1626) ~[spring-beans-4.3.25.RELEASE.jar:4.3.25.RELEASE]
	... 14 more
2020-06-17 19:31:01,283 DEBUG [scheduler_QuartzSchedulerThread] null core.QuartzSchedulerThread - batch acquisition of 1 triggers
2020-06-17 19:31:12,839 DEBUG [scheduler_QuartzSchedulerThread] null core.QuartzSchedulerThread - batch acquisition of 1 triggers
2020-06-17 19:31:12,839 DEBUG [scheduler_Worker-2] null core.JobRunShell - Calling execute on job IAF_Util.CheckReload
2020-06-17 19:31:12,840 INFO  [CheckReload[scheduler_Worker-2]] null scheduler.ConfiguredJob - Job [CheckReload] executing

**Comment by nielsm5** [comment]
Could you validate if the console is not updating, or if the configuration is not reloading?
I found a bug where old adapters would not disappear in certain cases, but the reload works.

**Comment by mhdirkse** [comment]
I think the config is not reloading. I did "DROP ALL OBJECTS" on the installation I presented in this issue. Then I uploaded my configuration again. I waited a few minutes to be sure that the auto reload should have happened. When back on the Adapter Status page, I still saw the error. I went to Test Pipeline and I could not select my adapter. It was not there.

I did a minor change to my uploaded config before uploading it again. I tested that it worked again by pressing the refresh button and trying it.
